### ISSUE TITLE
'Move to archive' function is broken

### ISSUE THREAD

**Issue by ags1234** [doublecmd/doublecmd#601]
When using 'Move to archive' function, DC shows an error saying function is not supported.
Is this a regression? Since I know this function worked in the past.
I am using build 10380, the latest as of today.

**Comment by alexx2000** [comment]
It depends on the archive type. Built-in `Zip` plugin does not support move operation and never supported it.
https://doublecmd.sourceforge.io/mantisbt/view.php?id=1559

**Comment by ags1234** [comment]
Indeed, I am referring to compressing files in zip format, with included plugin. Maybe I don't recall correctly when I say this function worked.
Can then the zip plugin be made to delete files after compressing them?
It's weird that DC can delete files and can compress them in zip, but cannot do this in one operation. Especially since it offers this function, and it promises to do so with the checkbox option.

[https://ibb.co/F0BZc9B](url)

[https://ibb.co/m8M4LFb](url)

**Comment by alexx2000** [comment]
> Can then the zip plugin be made to delete files after compressing them?

Move to archive really means compress and delete. I did not implement this feature because used library is too confused. So it is safer to not implement this function.

**Comment by PetbkA** [comment]
@alexx2000 
> Built-in Zip plugin does not support move operation and never supported it

Is this the same reason it doesn't support renaming files inside of archive? Nothing happens on rename hotkey

**Comment by alexx2000** [comment]
No, it is a limitation of [WCX-API](https://tinyurl.com/tcwcxapi221se) which does not have such function.

**Comment by ags1234** [comment]
> No, it is a limitation of [WCX-API](https://tinyurl.com/tcwcxapi221se) which does not have such function.

For anyone contributing to the source code, could you please take a look at this?
